docs/instrumentation.md

```diff
@@ -66,6 +66,14 @@ anycable_go_goroutines_num 5222
 # HELP anycable_go_disconnect_queue_size The size of delayed disconnect
 # TYPE anycable_go_disconnect_queue_size gauge
 anycable_go_disconnect_queue_size 0
+
+# HELP server_msg_total The total number of messages sent to clients
```

Comment:
Oh, it should have `anycable_go_` prefix.

Reply:
Good catch. I've fixed this now.

Reply:
@palkan Can I get approval on this? I've updated the description with my usecase.